.github/workflows/tests.yml

```diff
@@ -0,0 +1,230 @@
+        name: "Trigger premium primitives tests for python 3.6 release"
+        run: |
+          url=https://circleci.com/api/v1.1/project/github/FeatureLabs/premium-primitives/tree/main
+          curl -u ${{ secrets.PP_K }}: -d 'build_parameters[CIRCLE_JOB]=python_36_ft_release' $url
+
+  unit_tests:
+    name: ${{ matrix.python_version }} unit tests ${{ matrix.optional_libraries }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python_version: ["3.6", "3.7", "3.8"]
+        optional_libraries: ["minimal", "optional"]
+    steps:
+      - name: Set up python ${{ matrix.python_version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python_version }}
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Build featuretools package
+        run: make package_featuretools
+      - name: Set up pip and graphviz
+        run: |
+          pip config --site set global.progress_bar off
+          python -m pip install --upgrade pip
+          sudo apt update && sudo apt install -y graphviz
+      - if: ${{ matrix.optional_libraries != 'optional' }}
+        name: Install featuretools with test requirements (no optional requirements)
+        run: |
+          python -m pip install -e unpacked_sdist/
+          python -m pip install -r unpacked_sdist/test-requirements.txt
+      - if: ${{ matrix.optional_libraries == 'optional' }}
+        name: Install featuretools with test requirements and optional requirements
+        run: |
+          sudo apt install -y openjdk-11-jre-headless
+          JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
+          python -m pip install -e unpacked_sdist/[koalas]
+          python -m pip install -r unpacked_sdist/test-requirements.txt
+      - if: ${{ matrix.python_version != 3.6 || matrix.optional_libraries != 'optional' }}
+        name: Run unit tests (no code coverage)
+        run: |
+          cd unpacked_sdist
+          pytest featuretools/ -n 2
+      - if: ${{ matrix.python_version == 3.6 && matrix.optional_libraries == 'optional' }}
+        name: Run unit tests with code coverage
+        run: |
+          python -m pip install "$(cat dev-requirements.txt | grep codecov)"
+          coverage erase
+          cd unpacked_sdist/
+          coverage erase
+          pytest featuretools/ -n 2 --cov=featuretools --cov-config=../.coveragerc
+      - if: ${{ matrix.python_version == 3.6 && matrix.optional_libraries == 'optional' }}
+        name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          fail_ci_if_error: true
+
+  win_unit_tests:
+    name: ${{ matrix.python_version }} windows unit tests
+    runs-on: windows-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python_version: ["3.6", "3.7", "3.8"]
+    steps:
+      - name: Setup miniconda
+        uses: conda-incubator/setup-miniconda@v2.0.0
```

Comment:
should we use v2 instead of v2.0.0

Reply:
updated step to set up miniconda using original code